src/tasks/harbor/harbor.ts

```diff
@@ -190,58 +203,61 @@ async function ensureSystemSecret(): Promise<RobotSecret> {
       robotSecret = await createSystemRobotSecret()
     }
   }
-  setAuth(robotSecret.secret)
-  return robotSecret
+  bearerAuth.accessToken = robotSecret.secret
+  return bearerAuth
 }
 
-async function ensureProjectSecret(teamId: string, projectId: string): Promise<void> {
-  const namespace = `team-${teamId}`
-
-  let k8sSecret = (await getSecret(projectSecretName, namespace)) as RobotSecret
+/**
+ * Ensure that Harbor robot account and corresponding Kubernetes pull secret exist
```

Comment:
I see you're adding function docs, that's awesome, but do we use something like JSdoc or do you do it manually?

Reply:
It is added automatically while you start typing `/**`